src/Features/ERDDAP/Platform/Observations/CurrentConditions/wind.tsx

```diff
@@ -2,80 +2,177 @@
  * Wind specific current conditions card
  */
 import React from "react"
-import { Card, CardBody, CardHeader } from "reactstrap"
+import { Link } from "react-router-dom"
+import { Card, CardBody, CardHeader, Col } from "reactstrap"
 
 import { round } from "Shared/math"
 import { DataTimeSeries } from "Shared/timeSeries"
 import { compassDirection } from "Shared/unitConversion"
+import { useUnitSystem } from "Features/Units"
 import { UnitSystem } from "Features/Units/types"
 import { converter } from "Features/Units/Converter"
 
 import { WindTimeSeriesChart } from "components/Charts"
 
-import { PlatformDataset } from "../../../types"
+import { useDatasets } from "../../../hooks"
+import { PlatformFeature, PlatformTimeSeries } from "../../../types"
+import { pickWindDatasets, pickWindTimeSeries } from "../../../utils/wind"
+import { cardProps, observationLink } from "./common_card"
 
-interface Props {
-  datasets: PlatformDataset[]
-  unit_system: UnitSystem
+interface WindCardProps {
+  platform: PlatformFeature
 }
 
-export const WindCard: React.SFC<Props> = ({ datasets, unit_system }) => {
-  if (datasets.length < 1) {
-    return null
+/**
+ * If there are wind time series for a platform load and display recent data.
+ *
+ * @param platform Platform GeoJSON feature to display wind data from
+ */
+export const WindCard: React.FC<WindCardProps> = ({ platform }) => {
+  const aDayAgo = new Date()
+  aDayAgo.setDate(aDayAgo.getDate() - 1)
+
+  const { windTimeSeries, timeSeries } = pickWindTimeSeries(platform, aDayAgo)
+
+  // No current data or wind data don't exist for the platform
+  if (windTimeSeries.length < 1) {
+    const noWindTimeSeries = platform.properties.readings.filter((timeSeries) =>
+      timeSeries.data_type.standard_name.includes("wind")
+    )
+
+    if (noWindTimeSeries.length < 1) {
+      return null
+    }
+
+    return <OtherWindCard platform={platform}>No recent wind data available.</OtherWindCard>
+  }
+
+  if (timeSeries.length < 0) {
+    return <OtherWindCard platform={platform}>No wind data</OtherWindCard>
+  }
+
+  return <LoadWindCard platform={platform} timeSeries={timeSeries} />
+}
+
+interface LoadWindCardProps extends WindCardProps {
+  timeSeries: PlatformTimeSeries[]
+}
+
+/**
+ * Load data for wind time series, and connect currently selected unit_system
+ */
+const LoadWindCard: React.FC<LoadWindCardProps> = ({ platform, timeSeries }) => {
+  const unit_system = useUnitSystem()
```

Comment:
Codacy found an issue: [Identifier 'unit_system' is not in camel case.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6166252)